README.md

```diff
@@ -18,22 +24,22 @@ This project automates the setup of a development environment for working on Rub
 
 Building the virtual machine is this easy:
 
-    host $ git clone https://github.com/rails/rails-dev-box.git
-    host $ cd rails-dev-box
+    host $ git clone https://github.com/aidanharan/activerecord-sqlserver-adapter-dev-box.git
```

Comment:
This should be `git@github.com:rails-sqlserver/activerecord-sqlserver-adapter-dev-box.git` instead.

Reply:
Updated to `git clone https://github.com/rails-sqlserver/activerecord-sqlserver-adapter-dev-box.git` as that's the format used in the rails-dev-box.

```
~/dev$ git clone git@github.com:rails-sqlserver/activerecord-sqlserver-adapter-dev-box.git
Cloning into 'activerecord-sqlserver-adapter-dev-box'...
The authenticity of host 'github.com (140.82.118.4)' can't be established.
RSA key fingerprint is SHA256:nThbg6kXUpJWGl7E1IGOCspRomTxdCARLviKw6E5SY8.
Are you sure you want to continue connecting (yes/no/[fingerprint])?
```